apps/prairielearn/python/traverse.py

```diff
@@ -34,7 +34,7 @@
 def traverse_and_execute(
     html: str, fn: Callable[[lxml.html.HtmlElement], None]
 ) -> None:
-    elements = lxml.html.fragments_fromstring(html)
+    elements: list[lxml.html.HtmlElement] = lxml.html.fragments_fromstring(html)
```

Comment:
@eliotwrobson is there a better way to do this? The only possible return of a different type from this call is deleted in the following few lines.

Reply:
New code looks good. Piggyback that annotations in this file can be changed to all lowercase, but not really a huge deal as long as typechecking passes.